ReleaseProcess.md

```diff
@@ -24,62 +28,49 @@ The following is a step-by-step procedure for releasing a new version of Google-
 
 Important: When aborting a release / rolling back release preparations, make sure to also revert all changes to files which were done during the release (e.g. automatic replacement of version references).
 
-## Configuring a machine for deployment to Sonatype Repository
-
-This section was borrowed heavily from [Doclava release process](https://code.google.com/archive/p/doclava/wikis/ProcessRelease.wiki).
-
-1. Install/Configure GPG following this [guide](https://blog.sonatype.com/2010/01/how-to-generate-pgp-signatures-with-maven/).
-1. [Create encrypted passwords](https://maven.apache.org/guides/mini/guide-encryption.html).
-1. Create `~/.m2/settings.xml` similar to as described in [Doclava release process](https://code.google.com/p/doclava/wiki/ProcessRelease).
-1. Now for deploying a snapshot repository, use `mvn deploy`.
```

Comment:
Removed this because it is most likely pretty outdated / there is probably better or more up to date information available.